addons/mail/static/src/messaging/entity/follower/follower.js

```diff
@@ -36,14 +40,18 @@ function FollowerFactory({ Entity }) {
                 data2.isEditable = data.is_editable;
             }
             if ('partner_id' in data) {
-                const partnerData = { id: data.partner_id };
-                if ('email' in data) {
-                    partnerData.email = data.email;
-                }
-                if ('name' in data) {
-                    partnerData.name = data.name;
+                if (!data.partner_id) {
+                    data2.partne = [['unlink-all']];
+                } else {
+                    const partnerData = { id: data.partner_id };
+                    if ('email' in data) {
+                        partnerData.email = data.email;
+                    }
+                    if ('name' in data) {
+                        partnerData.name = data.name;
+                    }
```

Comment:
feels like this is the job of partner convertData

Reply:
don't know, was there before this commit ; but I can check